generate_protos.sh

```diff
@@ -84,6 +85,11 @@ rm -rf gen/pb-go/github.com
 mkdir -p gen/pb_python/validate
 cp -r validate/* gen/pb_python/validate/
 
+# Update the service code manually because the code generated by protoc is incorrect
+# More detail, check https://github.com/flyteorg/flyteidl/pull/303#discussion_r1002151053
+sed -i -e 's/protoReq.Id.ResourceType = ResourceType(e)/protoReq.Id.ResourceType = core.ResourceType(e)/g' gen/pb-go/flyteidl/service/admin.pb.gw.go
+rm -f gen/pb-go/flyteidl/service/admin.pb.gw.go-e
```

Comment:
cc @EngHabu Seems like there are some issues in the protoc. I can fix the error by updating it manually.
```bash
Error parsing file:  /Users/kevin/git/flyteidl/gen/pb-go/flyteidl/service/admin.pb.gw.go:2743:29: undeclared name: ResourceType
Error parsing file:  /Users/kevin/git/flyteidl/gen/pb-go/flyteidl/service/admin.pb.gw.go:2743:29: undeclared name: ResourceType
Error parsing file:  /Users/kevin/git/flyteidl/gen/pb-go/flyteidl/service/admin.pb.gw.go:2743:29: undeclared name: ResourceType
Error parsing file:  /Users/kevin/git/flyteidl/gen/pb-go/flyteidl/service/admin.pb.gw.go:2743:29: undeclared name: ResourceType
Error parsing file:  /Users/kevin/git/flyteidl/gen/pb-go/flyteidl/service/admin.pb.gw.go:2743:29: undeclared name: ResourceType
```
Other alternatives:
1. Bump protoc version (May need to update the image `lyft/protocgenerator:8167e11d3b3439373c2f033080a4b550078884a2`)
2. Add `DescriptionEntityGetRequest` in the IDL, which will be like the below. and then update the endpoint to `/api/v1/description_entities/{resource_type}/{id.project}/{id.domain}/{id.name}/{version}`
Note: protoc will fail to generate the code when using `id.resource_type` in the endpoint
```protobuf
message DescriptionEntityListRequest {
  // Identifies the specific type of resource that this identifier corresponds to.
  flyteidl.core.ResourceType resource_type  = 1;
  NamedEntityIdentifier id = 2;
  string version = 3
```

I can remove the change if you don't like it.

Reply:
Is there a plan to use the rest endpoint? if not, then maybe let's remove it (the REST endpoint) completely?

Reply:
yes, It will be used in flyteconsole